web-app/src/common/prediction.spec.ts

```diff
@@ -0,0 +1,116 @@
+import expect from 'expect';
+import { Coordinates, parseGameStateFromMultilineString } from './utils';
+import { getNextPlaySuggestion } from './prediction';
+
+function areCoordinatesEquals(coord1: Coordinates, coord2: Coordinates) {
+  return coord1.x === coord2.x && coord1.y === coord2.y;
+}
+
+describe('Should get an advice for the next play', () => {
+  it('Should get a good advice', () => {
+    const input = parseGameStateFromMultilineString(`
+  ⬡ ⬡ ⬡
+   ⬡ ⬡ ⬡
+    ⬢ ⬢ ⬡
+  `);
+
+    const nextPlayAdvice = getNextPlaySuggestion(input, "black");
+
+    expect(areCoordinatesEquals(nextPlayAdvice, { x: 2, y: 2 }) || areCoordinatesEquals(nextPlayAdvice, { x: 2, y: 1 })).toBeTruthy();
+  });
+
+  it('Should get a good advice', () => {
+    const input = parseGameStateFromMultilineString(`
+  ⬡ ⬡ ⬡
+   ⬡ ⬡ ⬡
+    ⬢ ⬡ ⬢
+  `);
+
+    const nextPlayAdvice = getNextPlaySuggestion(input, "black");
+
+    expect(areCoordinatesEquals(nextPlayAdvice, { x: 1, y: 2 }) || areCoordinatesEquals(nextPlayAdvice, { x: 1, y: 1 })).toBeTruthy();

```

Comment:
I disagree on this one. To me, playing at `{ x: 1, y: 1 }` doesn't make a connection, only `{ x: 1, y: 2 }` is valid